pkg/controller/plan/kubevirt.go

```diff
@@ -704,6 +738,13 @@ func (r *KubeVirt) virtualMachine(vm *plan.VMStatus) (object *cnv.VirtualMachine
 			vm.String())
 		object = r.emptyVm(vm)
 	}
+	//Add the original name and ID info to the VM annotations
+	if len(originalName) > 0 {
+		annotations := make(map[string]string)
+		annotations[AnnOriginalName] = originalName
+		annotations[AnnOriginalID] = vm.ID
+		object.ObjectMeta.Annotations = annotations
```

Comment:
wouldn't it override the annotations to have only AnnOriginalName and AnnOriginalID?

shouldn't it be something like:
```
annotations := object.ObjectMeta.Annotations
annotations[AnnOriginalName] = originalName
annotations[AnnOriginalID] = vm.ID
object.ObjectMeta.Annotations = annotations
```
(maybe nil check required for the first line, if it's nil then to make the map).

Reply:
At this point the annotation part does not exist, you can see in `emptyVm()` that it's not created yet, so here I am just initializing them, and later on in the flow, the rest of the annotations are added so there's no overriding (also been tested and all the annotation remains).